Gruntfile.js

```diff
@@ -323,7 +323,7 @@ module.exports = function (grunt) {
     });
 
     // task: install
-    grunt.registerTask('install', ['write-config', 'less', 'npm-install-extensions']);
+    grunt.registerTask('install', ['write-config', 'less', 'copy-browser-dependencies', 'npm-install-extensions']);
```

Comment:
Fast note: the script is referenced here as `copy-browser-dependencies` but is actually named `move-browser-dependencies`.